### ISSUE TITLE
homepage - build queries link tab focus height too long

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#64]
The height of the focus border when the Build Queries image/link has been tabbed to is too long covering the schedule tables. In the screenshot below, the bottom line hits the "Assemblies of God.." line on the table

<img width="1440" alt="Screen Shot 2020-12-07 at 11 07 39 AM" src="https://user-images.githubusercontent.com/618307/101374883-bc3ac380-387c-11eb-9478-6e9e23ace3c1.png">

**Comment by kimisgold** [comment]
This fix should be live and reading for testing.
